crates/solidity/inputs/language/src/lib.rs

```diff
@@ -5,6 +5,10 @@
 //! 2. Errors are reported only once, instead of repeating for every crate.
```

Comment:
I suggest compiling/serializing the grammar object once, same as we are doing with the old definition here. Check this comment for more details.

Reply:
But that prevents us adding imperative handlers to the types in the same crate as the DSL.

Reply:
Not sure why? it is the same serde type.

Reply:
Because the imperative handlers would go on the concrete types that the macro generates.

Reply:
Will need to think about this more. Resolving for now.